doc/changelog.rst

```diff
@@ -21,14 +21,20 @@ Description
 
 - Fix malformed logging format strings
 - Update linting support and apply to existing errors
+- Fix bug when user specify CPU and devices greater than 1
+- Fix bug when get_allocation called with reserved keywords
 
 Detailed Notes
 
 - Fix incorrectly formatted positional arguments in log format strings (PR330_)
 - Update pylint dependency, update .pylintrc, mitigate non-breaking issues, suppress api breaks (PR311_)
+- Add error when user requests CPU with devices greater than 1 within add_ml_model and add_script (PR324_)
+- Add error if user calls get_allocation with reserved keywords in slurm get_allocation (PR325_)
 
 .. _PR330: https://github.com/CrayLabs/SmartSim/pull/330
 .. _PR311: https://github.com/CrayLabs/SmartSim/pull/311
+.. _PR324: https://github.com/CrayLabs/SmartSim/pull/324
+.. _PR325: https://github.com/CrayLabs/SmartSim/pull/325
```

Comment:
It looks like I added my comments incorrectly and you followed suit. These should really be in reverse-chronological order. 

Could you please move yours and fix my mistake so these are in the order 330, 325, 324, 311 (both the details and descriptions)?